src/pages/lists/RecordTypesFilter/RecordTypesFilter.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { FC } from 'react';
+import {
+  Accordion,
+  FilterAccordionHeader
+} from '@folio/stripes/components';
+import { MultiSelectionFilter } from '@folio/stripes/smart-components';
+import { RECORD_TYPES_FILTER_KEY } from '../constants';
+import { EntityTypeSelectOption } from "../../../interfaces";
+
+type RecordTypesFilterProps = {
+  selectedRecordTypes: string[],
+  recordTypeConfig: {
+    label: React.ReactNode,
+    name: string,
+    values: {
+      value: string,
+      label: string
+    }[]
+  },
+  onChange: (selectedTypes: string[]) => void,
+  onClear: (group: string) => void
+}
+
+export const RecordTypesFilter: FC<RecordTypesFilterProps> = ({
+    selectedRecordTypes,
+    recordTypeConfig,
+    onChange,
+    onClear
+  }) => {
+  const filterEntityTypes = (a = '', options: EntityTypeSelectOption[]) => {
+    return {
+      renderedItems: options.filter(option => option.label.toLowerCase().includes(a.toLowerCase()) || '')
+    }
+  };
+
+  return (
+    <Accordion
+      label={recordTypeConfig.label}
+      separator={false}
+      // @ts-ignore
```

Comment:
Because types are created wrong. Accordion component passes all required props inside and then renders the Header, but he wants those props to be passed here

Reply:
If they're incorrect, please submit a PR to folio-org/stripes-types so the issue can be fixed